### DISCUSSION TITLE
FileUpload showing invalid file type for CSV file even if that file is CSV, problem only in FF

### DISCUSSION THREAD

**Discussion by vrjaroslav** [primefaces/community#1049] [PrimeNG]
FileUpload showing invalid file type for CSV file even if that file is CSV, problem only in FF 

Here is link: https://stackblitz.com/edit/gsnazh-wksafh?file=src%2Fapp%2Fdemo%2Ffile-upload-basic-demo.html

This is file for testing:
[testUploadFile.csv](https://github.com/primefaces/community/files/14082348/testUploadFile.csv)

Here is the issue: 
![testCSV](https://github.com/primefaces/community/assets/6131017/2a22ff98-b4d7-4e14-89f0-cc55c28a4d4d)

I have  FF of version: 122.0

**Comment by cetincakiroglu** [comment]
Hi,

I've tried in FF and couldn't replicate the issue, I've succeed to select CSV file

![Screenshot 2024-02-05 at 11 43 23](https://github.com/primefaces/community/assets/69278826/34c6cb46-842a-48a3-bc3c-22103c60aa64)
